Banking/src/main/java/com/example/Banking/repository/UserRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.example.Banking.repository;
+
+import com.example.Banking.model.Account;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<Account, Long> {
```

Comment:
Why we need this? We have an `AccountService`

Reply:
but account service can be accessed only when complete code of account is merged with master branch. since she is still working on account, i created this

Reply:
Just I said yesterday do not duplicate the efforts. First complete what is the dependent stuff and push it. I will review it right away since other work is dependent. But do not write same code in different PRs

Reply:
okay.